### PR TITLE
[System Audit] - Improved documentation

### PR DESCRIPTION
**Pull request by ShourieG**
## Type of change
Please label this PR with one of the following labels, depending on the scope of your change:
- Enhancement

## What does this PR do?
This PR improves the documentation a bit and clearly mentions that Windows systems are not supported at the moment.

## Checklist

- [x] I have reviewed [tips for building integrations](https://github.com/elastic/integrations/blob/main/docs/tips_for_building_integrations.md) and this pull request is aligned with them.
- [x] I have verified that all data streams collect metrics or logs.
- [x] I have added an entry to my package's `changelog.yml` file.
- [x] I have verified that Kibana version constraints are current according to [guidelines](https://github.com/elastic/elastic-package/blob/master/docs/howto/stack_version_support.md#when-to-update-the-condition).

### CODE REVIEW

File: packages/system_audit/changelog.yml [hunk 1]
```diff
@@ -1,3 +1,8 @@
+- version: "1.2.1"
+  changes:
+    - description: Improved the documentation and explicitly mentioned that Windows is not supported.
+      type: enhancement
```
Comment (bhapas) [inline packages/system_audit/changelog.yml:4]: bugfix?

**Review by bhapas — APPROVED** [review]
LGTM

File: packages/system_audit/changelog.yml (hunk 1 above)
Comment (ShourieG) [reply packages/system_audit/changelog.yml:4]: I mean there was nothing technically wrong with the documentation just that it was not clearly mentioned that windows is not supported, would that qualify as a bugfix ?

Comment (ShourieG) [reply packages/system_audit/changelog.yml:4]: Nvm, I can see other doc improvement PRs using a bugfix tag so will use that

Comment (bhapas) [reply packages/system_audit/changelog.yml]: To me , an `enhancement` is when a new feature is added and any correction a `bugfix`. May be I am wrong as well 😊

**Pull request merged**